src/contract/HandlerBasedContract.ts

```diff
@@ -688,7 +689,13 @@ export class HandlerBasedContract<State> implements Contract<State> {
     dummyTx.sortKey = await this._sorter.createSortKey(dummyTx.block.id, dummyTx.id, dummyTx.block.height, true);
     dummyTx.strict = strict;
     if (vrf) {
-      dummyTx.vrf = generateMockVrf(dummyTx.sortKey, arweave);
+      Arweave.utils;
```

Comment:
you can remove this line and then the `import Arweave`

Reply:
shit, wtf is this